ecommerce/ecommerce/settings.py

```diff
@@ -9,23 +9,46 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/3.2/ref/settings/
 """
+import environ
+import os
 
-from pathlib import Path
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, False)
```

Comment:
are you sure if debug param should be False by default?

Reply:
Yes. İt is stated in django docs that :
**Never deploy a site into production with DEBUG turned on.**.  In my opinion, default value should be false.
Thanks for review

Reply:
https://docs.djangoproject.com/en/3.2/ref/settings/